modules/yugabytedb/src/test/java/org/testcontainers/junit/yugabytedb/YugabyteDBYCQLTest.java

```diff
@@ -15,15 +15,24 @@ public class YugabyteDBYCQLTest {
 
     private static final String IMAGE_NAME = "yugabytedb/yugabyte:2.14.4.0-b26";
 
+    private static final String IMAGE_NAME_2_18 = "yugabytedb/yugabyte:2.18.3.0-b75";
+
     private static final DockerImageName YBDB_TEST_IMAGE = DockerImageName.parse(IMAGE_NAME);
 
     @Test
     public void testSmoke() {
+        doTestSmoke(IMAGE_NAME);
+    }
+
+    @Test
+    public void testSmoke2_18() {
+        doTestSmoke(IMAGE_NAME_2_18);
+    }
+
+    private void doTestSmoke(String image) {
```

Comment:
please, don't do this. It will break our docs by do not displaying the image version. See https://deploy-preview-7336--testcontainers.netlify.app/modules/databases/yugabytedb/#ycql-api

Instead, rollback this change and add the test at the end

```java
@Test
public void shouldStartWhenContainerIpIsUsedInWaitStrategy() {
    try (final YugabyteDBYCQLContainer ycqlContainer = new YugabyteDBYCQLContainer(IMAGE_NAME_2_18)) {
        ycqlContainer.start();
        assertThat(performQuery(ycqlContainer, "SELECT release_version FROM system.local").wasApplied()).isTrue();
    }
}
```

Reply:
btw, I couldn't submit changes to the branch because `main` branch was used. Next time, please, create a branch for the changes.